src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -910,6 +916,11 @@ export class Workbench implements IPartService {
 		this.workbenchLayout.layout();
 	}
 
+	private setFontAliasing(enabled: boolean) {
+		this.fontAliasingEnabled = enabled;
+		this.workbench.style('-webkit-font-smoothing', enabled ? 'antialiased' : '');
```

Comment:
What would be the correct approach here -- should it access body element from the workbench class by going up the dom tree or should i move the logic entirely to another class? As said above, i'm just getting familiar with the code -- suggestions are welcome.